sample-input/benchmarks/LRA/LRA.py

```diff
@@ -1,10 +1,13 @@
 import openmoc
+import openmoc.plotter as plotter
+from openmoc.options import Options
+from lattices import lattices, universes, cells
```

Comment:
Good catch, yes they can. I just kind of accumulated the list of items imported as I worked up from surfaces to cells to universes to lattices to the main input file.